translations/translations.json

```diff
@@ -7902,7 +7902,11 @@
       "alphabetLetters": "A-Z",
       "clearFilter": "Filter {{filter}} löschen",
       "showOwnLocation": "Eigenen Standort zeigen",
-      "makeAppointment": "Termin vereinbaren"
+      "makeAppointment": "Termin vereinbaren",
+      "appointmentNecessary": "Termin erforderlich",
+      "makeAppointmentTooltipWithLink": "Vereinbare an diesem Tag einen Termin über <1>die Webseite</1> oder telefonisch.",
```

Comment:
Nope, because of capitalisation. Doesn't work in e.g. English where it would incorrectly say "Make an appointment [...] via the Website", so I decided to also include the article in the link to make it clearer for translators.

About linking to the phone number, could do, it wasn't in the design so I didn't. I'm also not sure it's great for usability because the two links would be pretty close together on small screens, so someone trying to get to the website might end up calling someone or vice versa.

Reply:
Makes sense, thanks. Hmm, I think we could use links for both, clicking a phone number doesn't directly call someone, it just opens the phone app. I think the risk is very low with mostly allowing better UX.

Reply:
Hmm, I don't like the phone number thing in here. I'll take a tie breaker vote @f1sh1918 @ztefanie :)

Reply:
i would prefer keeping this separate. there is a separate url to make an appointment. The phone number is clickable in the contact details. should be enough